src/commands/branches.ts

```diff
@@ -186,7 +187,14 @@ export const builder = (argv: yargs.Argv) =>
       'Get a branch',
       (yargs) => yargs,
       async (args) => await get(args as any),
-    );
+    )
+    .command({
+      command: 'schema-diff [branch] <compare-branch>',
```

Comment:
I think square brackets are of optional args, and optional shouldn't precede required

Reply:
yea, ok, I admit your position is stronger :) 👍